ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,658 @@
+                    breadcrumbsList.push([currentLevelNumber, level.pointOptions]);
+                }
+            });
+        }
+        this.breadcrumbsList = breadcrumbsList;
+    }
+
+    /**
+     * Destroy the chosen buttons and separators.
+     *
+     * @requires  modules/drilldown
+     *
+     * @function Highcharts.Chart#destroy
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     * @param {Highcharts.Breadcrumbs} lastVisibleLevel
+     *        Number of levels to destoy.
+     */
+    public destroy(this: Breadcrumbs, lastVisibleLevel: number): void {
+        const breadcrumbs = this,
+            chart = breadcrumbs.chart,
+            breadcrumbsList = breadcrumbs.breadcrumbsList,
+            initialSeriesLevel = -1,
+            breadcrumbsOptions = chart.options.drilldown?.breadcrumbs as Highcharts.BreadcrumbsOptions;
+
+        // Click the main breadcrumb
+        if (lastVisibleLevel === initialSeriesLevel) {
+            breadcrumbsList.forEach(function (el): void {
+                const button = el[2],
+                    conector = el[3];
+
+                // Remove SVG elements fromt the DOM.
+                button ? button.destroy() : void 0;
+                conector ? conector.destroy() : void 0;
+            });
+            // Clear the breadcrums list array.
+            breadcrumbsList.length = 0;
+        } else {
+            const prevConector = breadcrumbsList[lastVisibleLevel + 1][3];
+
+            // Remove connector from the previous button.
+            prevConector ? prevConector.destroy() : void 0;
+            breadcrumbsList[lastVisibleLevel + 1].length = 3;
+
+            for (let i = lastVisibleLevel + 2; i < breadcrumbsList.length; i++) {
+                const el = breadcrumbsList[i],
+                    button = el[2],
+                    conector = el[3];
+
+                // Remove SVG elements fromt the DOM.
+                button ? button.destroy() : void 0;
+                conector ? conector.destroy() : void 0;
+            }
+            // Clear the breadcrums list array.
+            breadcrumbsList.length = lastVisibleLevel + 2;
+        }
+
+        if (breadcrumbsOptions.position && breadcrumbsOptions.position.align !== 'left') {
+            breadcrumbs.alignGroup();
+        }
+    }
+
+    /**
+     * Create a group, then draw breadcrumbs together with the separators.
+     *
+     * @requires  modules/drilldown
+     *
+     * @function Highcharts.Chart#draw
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     */
+    public draw(this: Breadcrumbs): void {
+        const breadcrumbs = this,
+            chart = breadcrumbs.chart,
+            breadcrumbsList = breadcrumbs.breadcrumbsList,
+            breadcrumbsOptions = chart.options.drilldown?.breadcrumbs,
+            lastBreadcrumbs = breadcrumbsList[breadcrumbsList.length - 2][2],
+            buttonPadding = 5;
```

Comment:
should it not be configurable?

Reply:
It was not in the docs, but I don't see any reason why it shouldn't be configurable.